docs/integrations/creating-nodes/build/n8n-node.md

```diff
@@ -0,0 +1,186 @@
+---
+#https://www.notion.so/n8n/Frontmatter-432c2b8dff1f43d4b1c8d20075510fe4
+contentType: howto
+search:
+  boost: 3
+---
+
+# Using the n8n-node tool
+
+The `n8n-node` tool is the official CLI for developing community nodes for n8n. You can use it to scaffold out new nodes, build your projects, and run your node as you develop it.
+
+Using `n8n-node`, you can create nodes that adhere to the 
+
+## Prerequisites
+
+Before you can use the `n8n-node` tool, you must [install n8n locally with `npm`](/hosting/installation/npm.md). The `n8n-node` tool runs the local n8n instance with your community node loaded to provide live previews.
+
+## Get n8n-node
+
+### Run n8n-node without installing
+
+You can run `n8n-node` directly without installing with `npx`. For example, to view the command's help information, you can type:
+
+```shell
+npx --package=@n8n/node-cli n8n-node --help
```

Comment:
This explanation is really good, but could we align with the new README? https://www.npmjs.com/package/@n8n/node-cli?activeTab=readme

I'm trying to simplify the dev flow to:

```
npm create @n8n/node@latest

cd n8n-nodes-example
npm run dev # and other commands, using the CLI as a local dependency
```

without requiring explicit installation

Reply:
Yeah, for sure. I'll test this out and update to try to align with the new preferred flow.

Reply:
I know the README uses `pnpm` commands at the moment but I will change it to `npm` in an upcoming version so we can be consistent 😅